src/main/java/com/offz/spigot/staminaclimb/config/StaminaConfig.kt

```diff
@@ -13,7 +13,10 @@ object StaminaConfig : IdofrontConfig<StaminaConfig.Data>(staminaClimb, Data.ser
         val staminaRegenInAir: Double,
         val staminaRemovePerTick: Double,
         val staminaRemoveWhileMoving: Double,
-        val barRed: Double,
+        val barMin: Double,
```

Comment:
I'm not sure about renaming this, barRed is just the red area which will be greater than 0, barMin sounds like the absolute minimum it will ever reach. I got confused by this at first.

Reply:
How about renaming it to barRedZone? I still think barRed seems a bit odd since you would think it would refer to a color or something, but I agree that barMin isn't that good either.